src/app/app.component.ts

```diff
@@ -19,9 +20,10 @@ export class AppComponent implements OnInit {
   displayAnswer: boolean = false;
   answerStatus: string = 'Show';
 
-  ml_questions: any = JSON.parse(JSON.stringify(questions));
+  ml_questions: any = JSON.parse(JSON.stringify(ml_questions));
   finance_questions: any = JSON.parse(JSON.stringify(finance_questions));
-  questions: Array<any> = [this.ml_questions.concat(this.finance_questions), finance_questions, questions, []]
+  python_questions: any = JSON.parse(JSON.stringify(python_questions));
+  questions: Array<any> = [this.ml_questions.concat(this.finance_questions.concat(this.python_questions), finance_questions, ml_questions, python_questions, []]
```

Comment:
Missing `)` to close `concat` at column 105 leads to a build error
```suggestion
  questions: Array<any> = [this.ml_questions.concat(this.finance_questions.concat(this.python_questions)), finance_questions, ml_questions, python_questions, []]
```